geoalchemy2/functions.py

```diff
@@ -312,6 +312,18 @@ def __init__(self, *args, **kwargs):
      'Returns a geometry that represents the shared portion of geomA and '
      'geomB. The geography implementation does a transform to geometry to do '
      'the intersection and then transform back to WGS84.'),
+   
+    ('ST_Line_Locate_Point', types.Geometry,
+     'Returns a locate point from geometryA into line.'),
```

Comment:
The name should be `ST_LineLocatePoint` as in PostGIS. And please use the same description as the one in the PostGIS doc. See http://postgis.net/docs/ST_LineLocatePoint.html.